example/android/app/src/main/AndroidManifest.xml

```diff
@@ -23,7 +23,7 @@
         </activity>
 
         <service
```

Comment:
Following up with my other comment, let's remove this line in favour of having the Manifest at the RN lib level.